docs/database-classes.md

```diff
@@ -277,8 +277,41 @@ transactor.Transact(db =>
 });
 ```
 
+## Discovery
+
+Starcounter automatically discovers all database classes in the application assembly, which does not include all referenced assemblies. Manual configuration is required to use database classes from referenced assemblies.
+
+### Referenced assembly
+
+```cs
+[Database]
+public abstract class Something
+{
+}
+```
+
+### Application assembly
+
+```cs
+using var services = new ServiceCollection()
+    .AddStarcounter($"Database=./.database/test;OpenMode=CreateIfNotExists;StartMode=StartIfNotRunning;StopMode=IfWeStarted")
+    .Configure<Starcounter.Database.Binding.TypeBindingOptions>(o =>
+    {
+        Type[] extraDatabaseTypes = typeof(Something) // Getting type of a database class.
+            .Assembly // Getting assembly which defines the type.
+            .GetTypes() // Getting all types in the assembly.
+            .Where(x => x.GetCustomAttributes(typeof(DatabaseAttribute), true).Any()) // Filtering the types by the `[Database]` attribute.
```

Comment:
```suggestion
            .Where(x => x.GetCustomAttribute<DatabaseAttribute>(true) != null) // Filtering the types by the `DatabaseAttribute` attribute.
```

I would prefer the generic method finding the first decoration, and comparing with null.